packages/components/src/item-group/styles.ts

```diff
@@ -42,7 +41,7 @@ export const separated = css`
 		border-bottom: 1px solid ${ CONFIG.surfaceBorderColor };
 	}
 
-	> *:last-of-type:not( :focus ) {
+	> *:last-child:not( :focus ) {
```

Comment:
I know that `last-child` and `first-child` cause warnings in Storybook, but using `last-of-type` and `first-of-type` was causing an issue when mixing and matching different HTML elements for the `Item`s.

THe most frequent example was when mixing Items with `isAction={true}` and Items with `isAction={false}`, since they render `button` and `divs`